cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -723,7 +723,8 @@ public MapAndStatus(T loadMap, K status) {
                                                     KafkaRebalance kafkaRebalance,
                                                     KafkaRebalanceAnnotation rebalanceAnnotation,
                                                     RebalanceOptions.RebalanceOptionsBuilder rebalanceOptionsBuilder) {
-        if (rebalanceAnnotation == KafkaRebalanceAnnotation.refresh) {
+
+        if (shouldRenewRebalance(kafkaRebalance, rebalanceAnnotation)) {
             // The user has fixed the error on the resource and want to 'refresh'
```

Comment:
Update this comment with the options for CC being activated when it was previously deactivated.